RestaurantManagementSystem/RestaurantManagementSystem/src/Core/Contracts/IEmailService.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace RestaurantManagementSystem.src.Core.Contracts
```

Comment:
what about `SendVerificationEmail(ContactInfo requestContactinfo)`

Reply:
it was like that but then it thought  this method can send any type of emails so why i just make it specified in to  send verification email 
for example when the admin approves the restaurant i can use it to send an email to tell the user "Welcome To Local Eats"

Reply:
Why do we need a function that sends all types of emails? It would be cleaner to have two functions, both internally can re-use a private base function.